modules/swagger-codegen/src/main/resources/lua/.travis.yml

```diff
@@ -1,8 +1,6 @@
-language: go
+language: lua
```

Comment:
Sadly travis doesn't support lua. Us lua devs use a tool 'hererocks' to run in travis
https://github.com/mpeterv/hererocks/#using-hererocks-to-set-up-automated-testing